src/main/java/de/cas/mse/exercise/diamond/Diamond.java

```diff
@@ -4,28 +4,28 @@
 
 public class Diamond {
 
-	public String print(int n) {
-		if (n <= 0 || n % 2 == 0) {
+	public String printDiamond(int diamondHeight) {
```

Comment:
Since the class is already called `Diamond`, there is no need to use the term diamon again.

Reply:
yes youre right